### DISCUSSION TITLE
Removed the need to review each PR before it goes to Vercel!

### DISCUSSION THREAD

**Discussion by ykdojo** [ykdojo/defaang#111] [Announcements]
I just removed the need to review each PR before it's deployed to Vercel!

How?

By setting up an additional organization/database in Supabase.

So we have two databases there now - one for production, and the other one for development.

Only the production branch (the main one) uses the production DB, and every other branch uses the dev one.

**Comment by iShibi** [comment]
Thank you so much 🙏

**Comment by anaskhan28** [comment]
Godd work because of this now non production stuffs are PR separately
